Jenkinsfile

```diff
@@ -410,15 +474,26 @@ EOF"""
           }
 
           if (params.IS_SMPE_PACKAGE) {
-            // FIXME: add parameters to ./install-SMPE-PAX.sh sript
-            // FIXME: may need to "mv zowe.pax zowe.pax.Z" if SMP/e package is compressed.
+            smpePathZfs = "${params.INSTALL_DIR}/zowe/smpe"
             sh """SSHPASS=${PASSWORD} sshpass -e ssh -tt -o StrictHostKeyChecking=no -o PubkeyAuthentication=no -p ${params.TEST_IMAGE_GUEST_SSH_PORT} ${USERNAME}@${params.TEST_IMAGE_GUEST_SSH_HOST} << EOF
 cd ${params.INSTALL_DIR} && \
   (iconv -f ISO8859-1 -t IBM-1047 install-SMPE-PAX.sh > install-SMPE-PAX.sh.new) && mv install-SMPE-PAX.sh.new install-SMPE-PAX.sh && chmod +x install-SMPE-PAX.sh
-./install-SMPE-PAX.sh ${params.INSTALL_DIR}/zowe.pax || { echo "[install-SMPE-PAX.sh] failed"; exit 1; }
-echo "[install-SMPE-PAX.sh] succeeds" && exit 0
+./install-SMPE-PAX.sh \
+  ${smpeHlq} \
+  ${smpeHlqCsi} \
+  ${smpeHlqTzone} \
+  ${smpeHlqDzone} \
+  ${smpePathPrefix} \
+  ${params.INSTALL_DIR} \
+  ${smpePathZfs} \
+  ${smpeFmid} \
+  ${smpeRelfilePrefix} || { echo "[install-SMPE-PAX.sh] failed"; exit 1; }
+echo "[install-SMPE-PAX.sh] done, start configuring ..."
+. ${smpePathPrefix}/scripts/configure/zowe-configure.sh
```

Comment:
@John-A-Davies is this the correct way to launch `zowe-configure.sh`?

Reply:
Basically yes.  On line 482, Note that the first parm is the HLQ of datasets, but not just <HLQ>.SMPE.**, it's also the HLQ for <HLQ>.<FMID>.F1 etc.  I only mention this because you call that parm "smpeHlq".

Line 492 looks like you're doing a `source` command.  Perhaps there should be a "/" between the "." and the "$"  ?